releasenotes/notes/properties-last-update-date-3d09884ff8634a2f.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    The `last_update_date` parameter for
+    :class:`~qiskit.providers.models.BackendProperties` can now be either a
+    ``datetime`` object or a datetime string (in ISO format).
+upgrade:
+  - |
+    A new package, `python-dateutil`, is now required and has been added to
```

can you make this a link to the pypi package:

```suggestion
    A new package, `python-dateutil <https://pypi.org/project/python-dateutil/>`_, is now required and has been added to
```